odh/overlays/moc/monitoring/servicemonitors/observatorium-servicemonitor.yaml

```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: observatorium-monitor
+  labels:
+    app: prometheus
```

Comment:
Didn't @hemajv use different labels for her monitors?

Reply:
are you referring to were the ones I added to the [prometheus alerting rules](https://github.com/operate-first/apps/blob/master/odh/base/monitoring/overrides/prometheus-operator/overlays/alerts/prometheus-rules.yaml#L4)? because I didn't add any service monitors

Reply:
ah yes that's the one. I feel like we should use similar/same labels

Reply:
what label do you suggest @anishasthana ?

Reply:
`    prometheus: odh-monitoring`

Reply:
updated :+1: